app/controllers/projects_controller.rb

```diff
@@ -1,21 +1,22 @@
 class ProjectsController < ApplicationController
   def index
-    @projects = current_user.organization&.projects
-    redirect_to new_project_path unless @projects.present?
+    redirect_to new_organization_project_path(organization) unless projects.present?
```

Comment:
The idea was to use memorization... And because a project will be always present if an organization is present, I move that logic to the projects method